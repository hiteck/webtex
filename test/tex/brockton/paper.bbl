\begin{thebibliography}{46}
\providecommand\natexlab[1]{#1}
\providecommand\JournalTitle[1]{#1}

\bibitem[{{Benz} \& {G\"{u}del}(1994)}]{bg94}
{Benz}, A.~O., \& {G\"{u}del}, M. 1994,
  \href{http://adsabs.harvard.edu/abs/1994A\%26A...285..621B}{\JournalTitle{A\&A},
  285, 621}

\bibitem[{{Berger}(2002)}]{b02}
{Berger}, E. 2002, \href{http://dx.doi.org/10.1086/340301}{\JournalTitle{ApJ},
  572, 503}

\bibitem[{{Berger}(2006)}]{b06b}
---. 2006, \href{http://dx.doi.org/10.1086/505787}{\JournalTitle{ApJ}, 648,
  629}

\bibitem[{{Berger} {et~al.}(2001){Berger}, {Ball}, {Becker}, {Clarke}, {Frail},
  {Fukuda}, {Hoffman}, {Mellon}, {Momjian}, {Murphy}, {Teng}, {Woodruff},
  {Zauderer}, \& {Zavala}}]{bbb+01}
{Berger}, E., {Ball}, S., {Becker}, K.~M., {et~al.} 2001,
  \href{http://dx.doi.org/10.1038/35066514}{\JournalTitle{Nature}, 410, 338}

\bibitem[{{Berger} {et~al.}(2005){Berger}, {Rutledge}, {Reid}, {Bildsten},
  {Gizis}, {Liebert}, {Mart\'{\i}n}, {Basri}, {Jayawardhana}, {Brandeker},
  {Fleming}, {Johns-Krull}, {Giampapa}, {Hawley}, \& {Schmitt}}]{brr+05}
{Berger}, E., {Rutledge}, R.~E., {Reid}, I.~N., {et~al.} 2005,
  \href{http://dx.doi.org/10.1086/430343}{\JournalTitle{ApJ}, 627, 960}

\bibitem[{{Berger} {et~al.}(2008){Berger}, {Gizis}, {Giampapa}, {Rutledge},
  {Liebert}, {Mart\'{\i}n}, {Basri}, {Fleming}, {Johns-Krull}, {Phan-Bao}, \&
  {Sherry}}]{bgg+08}
{Berger}, E., {Gizis}, J.~E., {Giampapa}, M.~S., {et~al.} 2008,
  \href{http://dx.doi.org/10.1086/524769}{\JournalTitle{ApJ}, 673, 1080}

\bibitem[{{Berger} {et~al.}(2009){Berger}, {Rutledge}, {Phan-Bao}, {Basri},
  {Giampapa}, {Gizis}, {Liebert}, {Mart\'{\i}n}, \& {Fleming}}]{brpb+09}
{Berger}, E., {Rutledge}, R.~E., {Phan-Bao}, N., {et~al.} 2009,
  \href{http://dx.doi.org/10.1088/0004-637x/695/1/310}{\JournalTitle{ApJ}, 695,
  310}

\bibitem[{{Berger} {et~al.}(2010){Berger}, {Basri}, {Fleming}, {Giampapa},
  {Gizis}, {Liebert}, {Mart\'{\i}n}, {Phan-Bao}, \& {Rutledge}}]{bbf+10}
{Berger}, E., {Basri}, G., {Fleming}, T.~A., {et~al.} 2010,
  \href{http://dx.doi.org/10.1088/0004-637x/709/1/332}{\JournalTitle{ApJ}, 709,
  332}

\bibitem[{{Browning}(2008)}]{b08}
{Browning}, M.~K. 2008,
  \href{http://dx.doi.org/10.1086/527432}{\JournalTitle{ApJ}, 676, 1262}

\bibitem[{{Burgasser} {et~al.}(2013){Burgasser}, {Melis}, {Zauderer}, \&
  {Berger}}]{bmzb13}
{Burgasser}, A., {Melis}, C., {Zauderer}, A., \& {Berger}, E. 2013,
  \href{http://dx.doi.org/10.1088/2041-8205/762/1/L3}{\JournalTitle{ApJL}, 762,
  L3}

\bibitem[{{Burgasser} \& {Putman}(2005)}]{bp05}
{Burgasser}, A., \& {Putman}, M. 2005,
  \href{http://dx.doi.org/10.1086/429788}{\JournalTitle{ApJ}, 626, 486}

\bibitem[{{Burgasser} {et~al.}(2006){Burgasser}, {Geballe}, {Leggett},
  {Kirkpatrick}, \& {Golimowski}}]{bgl+06}
{Burgasser}, A.~J., {Geballe}, T.~R., {Leggett}, S.~K., {Kirkpatrick}, J.~D.,
  \& {Golimowski}, D.~A. 2006,
  \href{http://dx.doi.org/10.1086/498563}{\JournalTitle{ApJ}, 637, 1067}

\bibitem[{{Burgasser} {et~al.}(2003{\natexlab{a}}){Burgasser}, {Kirkpatrick},
  {Liebert}, \& {Burrows}}]{bklb03}
{Burgasser}, A.~J., {Kirkpatrick}, J.~D., {Liebert}, J., \& {Burrows}, A.
  2003{\natexlab{a}},
  \href{http://dx.doi.org/10.1086/376756}{\JournalTitle{ApJ}, 594, 510}

\bibitem[{{Burgasser} {et~al.}(2003{\natexlab{b}}){Burgasser}, {Kirkpatrick},
  {Reid}, {Brown}, {Miskey}, \& {Gizis}}]{bkr+03}
{Burgasser}, A.~J., {Kirkpatrick}, J.~D., {Reid}, I.~N., {et~al.}
  2003{\natexlab{b}},
  \href{http://dx.doi.org/10.1086/346263}{\JournalTitle{ApJ}, 586, 512}

\bibitem[{{Burgasser} {et~al.}(2011){Burgasser}, {Sitarski}, {Gelino},
  {Logsdon}, \& {Perrin}}]{bsg+11}
{Burgasser}, A.~J., {Sitarski}, B.~N., {Gelino}, C.~R., {Logsdon}, S.~E., \&
  {Perrin}, M.~D. 2011,
  \href{http://dx.doi.org/10.1088/0004-637x/739/1/49}{\JournalTitle{ApJ}, 739,
  49}

\bibitem[{{Burgasser} {et~al.}(1999){Burgasser}, {Kirkpatrick}, {Brown},
  {Reid}, {Gizis}, {Dahn}, {Monet}, {Beichman}, {Liebert}, {Cutri}, \&
  {Skrutskie}}]{bkb+99}
{Burgasser}, A.~J., {Kirkpatrick}, J.~D., {Brown}, M.~E., {et~al.} 1999,
  \href{http://dx.doi.org/10.1086/312221}{\JournalTitle{ApJL}, 522, L65}

\bibitem[{{Carson} {et~al.}(2011){Carson}, {Marengo}, {Patten}, {Luhman},
  {Sonnett}, {Hora}, {Schuster}, {Allen}, {Fazio}, {Stauffer}, \&
  {Schnupp}}]{cmp+11}
{Carson}, J.~C., {Marengo}, M., {Patten}, B.~M., {et~al.} 2011,
  \href{http://dx.doi.org/10.1088/0004-637x/743/2/141}{\JournalTitle{ApJ}, 743,
  141}

\bibitem[{{Chabrier} \& {K\"{u}ker}(2006)}]{ck06}
{Chabrier}, G., \& {K\"{u}ker}, M. 2006,
  \href{http://dx.doi.org/10.1051/0004-6361:20042475}{\JournalTitle{A\&A}, 446,
  1027}

\bibitem[{{Condon}(2002)}]{c02}
{Condon}, J.~J. 2002,
  \href{http://adsabs.harvard.edu/cgi-bin/nph-bib_query?bibcode=2002ASPC..278..155C}{\JournalTitle{ASP~Conf.~Ser.},
  278, 155}

\bibitem[{{Dobler} {et~al.}(2006){Dobler}, {Stix}, \& {Brandenburg}}]{dsb06}
{Dobler}, W., {Stix}, M., \& {Brandenburg}, A. 2006,
  \href{http://dx.doi.org/10.1086/498634}{\JournalTitle{ApJ}, 638, 336}

\bibitem[{{Donati} {et~al.}(2008){Donati}, {Morin}, {Petit}, {Delfosse},
  {Forveille}, {Auri\`{e}re}, {Cabanac}, {Dintrans}, {Fares}, {Gastine},
  {Jardine}, {Ligni\`{e}res}, {Paletou}, {Ramirez Velez}, \&
  {Th\'{e}ado}}]{dmp+08}
{Donati}, J.~F., {Morin}, J., {Petit}, P., {et~al.} 2008,
  \href{http://dx.doi.org/10.1111/j.1365-2966.2008.13799.x}{\JournalTitle{MNRAS},
  390, 545}

\bibitem[{{Dulk}(1985)}]{d85}
{Dulk}, G.~A. 1985,
  \href{http://dx.doi.org/10.1146/annurev.aa.23.090185.001125}{\JournalTitle{ARA\&A},
  23, 169}

\bibitem[{{Durney} {et~al.}(1993){Durney}, {de Young}, \& {Roxburgh}}]{ddyr93}
{Durney}, B.~R., {de Young}, D.~S., \& {Roxburgh}, I.~W. 1993,
  \href{http://dx.doi.org/10.1007/bf00690652}{\JournalTitle{Solar Physics},
  145, 207}

\bibitem[{{Fomalont} {et~al.}(1991){Fomalont}, {Windhorst}, {Kristian}, \&
  {Kellerman}}]{fwkk91}
{Fomalont}, E.~B., {Windhorst}, R.~A., {Kristian}, J.~A., \& {Kellerman}, K.~I.
  1991, \href{http://dx.doi.org/10.1086/115952}{\JournalTitle{AJ}, 102, 1258}

\bibitem[{{Frail} {et~al.}(2012){Frail}, {Kulkarni}, {Ofek}, {Bower}, \&
  {Nakar}}]{fko+12}
{Frail}, D.~A., {Kulkarni}, S.~R., {Ofek}, E.~O., {Bower}, G.~C., \& {Nakar},
  E. 2012,
  \href{http://dx.doi.org/10.1088/0004-637X/747/1/70}{\JournalTitle{ApJ}, 747,
  70}

\bibitem[{{Gizis} {et~al.}(2000){Gizis}, {Monet}, {Reid}, {Kirkpatrick},
  {Liebert}, \& {Williams}}]{gmr+00}
{Gizis}, J., {Monet}, D., {Reid}, N., {et~al.} 2000,
  \href{http://dx.doi.org/10.1086/301456}{\JournalTitle{AJ}, 120, 1085}

\bibitem[{{G\"{u}del} \& {Benz}(1993)}]{gb93}
{G\"{u}del}, M., \& {Benz}, A. 1993,
  \href{http://dx.doi.org/10.1086/186766}{\JournalTitle{ApJ}, 405, L63}

\bibitem[{{Hall}(2002)}]{h02b}
{Hall}, P.~B. 2002,
  \href{http://dx.doi.org/10.1086/339020}{\JournalTitle{ApJL}, 564, L89}

\bibitem[{{Hallinan} {et~al.}(2006){Hallinan}, {Antonova}, {Doyle}, {Bourke},
  {Brisken}, \& {Golden}}]{had+06}
{Hallinan}, G., {Antonova}, A., {Doyle}, J.~G., {et~al.} 2006,
  \href{http://dx.doi.org/10.1086/508678}{\JournalTitle{ApJ}, 653, 690}

\bibitem[{{Hallinan} {et~al.}(2008){Hallinan}, {Antonova}, {Doyle}, {Bourke},
  {Lane}, \& {Golden}}]{had+08}
---. 2008, \href{http://dx.doi.org/10.1086/590360}{\JournalTitle{ApJ}, 684,
  644}

\bibitem[{{Hallinan} {et~al.}(2007){Hallinan}, {Bourke}, {Lane}, {Antonova},
  {Zavala}, {Brisken}, {Boyle}, {Vrba}, {Doyle}, \& {Golden}}]{hbl+07}
{Hallinan}, G., {Bourke}, S., {Lane}, C., {et~al.} 2007,
  \href{http://dx.doi.org/10.1086/519790}{\JournalTitle{ApJL}, 663, L25}

\bibitem[{{McLean} {et~al.}(2012){McLean}, {Berger}, \& {Reiners}}]{mbr12}
{McLean}, M., {Berger}, E., \& {Reiners}, A. 2012,
  \href{http://dx.doi.org/10.1088/0004-637x/746/1/23}{\JournalTitle{ApJ}, 746,
  23}

\bibitem[{{McMullin} {et~al.}(2007){McMullin}, {Waters}, {Schiebel}, {Young},
  \& {Golap}}]{thecasa}
{McMullin}, J.~P., {Waters}, B., {Schiebel}, D., {Young}, W., \& {Golap}, K.
  2007,
  \href{http://adsabs.harvard.edu/cgi-bin/nph-bib_query?bibcode=2007ASPC..376..127M}{\JournalTitle{ASP~Conf.~Ser.},
  376, 127}

\bibitem[{{Mohanty} {et~al.}(2002){Mohanty}, {Basri}, {Shu}, {Allard}, \&
  {Chabrier}}]{mbs+02}
{Mohanty}, S., {Basri}, G., {Shu}, F., {Allard}, F., \& {Chabrier}, G. 2002,
  \href{http://dx.doi.org/10.1086/339911}{\JournalTitle{ApJ}, 571, 469}

\bibitem[{{Morin} {et~al.}(2008){Morin}, {Donati}, {Petit}, {Delfosse},
  {Forveille}, {Albert}, {Auri\`{e}re}, {Cabanac}, {Dintrans}, {Fares},
  {Gastine}, {Jardine}, {Ligni\`{e}res}, {Paletou}, {Ramirez Velez}, \&
  {Th\'{e}ado}}]{mdp+08}
{Morin}, J., {Donati}, J.~F., {Petit}, P., {et~al.} 2008,
  \href{http://dx.doi.org/10.1111/j.1365-2966.2008.13809.x}{\JournalTitle{MNRAS},
  390, 567}

\bibitem[{{Reiners} \& {Basri}(2006)}]{rb06}
{Reiners}, A., \& {Basri}, G. 2006,
  \href{http://dx.doi.org/10.1086/503324}{\JournalTitle{ApJ}, 644, 497}

\bibitem[{{Reiners} \& {Basri}(2007)}]{rb07}
---. 2007, \href{http://dx.doi.org/10.1086/510304}{\JournalTitle{ApJ}, 656,
  1121}

\bibitem[{{Route} \& {Wolszczan}(2012)}]{rw12}
{Route}, M., \& {Wolszczan}, A. 2012,
  \href{http://dx.doi.org/10.1088/2041-8205/747/2/l22}{\JournalTitle{ApJL},
  747, L22}

\bibitem[{{Rutledge} {et~al.}(2000){Rutledge}, {Basri}, {Mart\'{\i}n}, \&
  {Bildsten}}]{rbmb00}
{Rutledge}, R.~E., {Basri}, G., {Mart\'{\i}n}, E.~L., \& {Bildsten}, L. 2000,
  \href{http://dx.doi.org/10.1086/312817}{\JournalTitle{ApJL}, 538, L141}

\bibitem[{{Sault} \& {Wieringa}(1994)}]{themfs}
{Sault}, R.~J., \& {Wieringa}, M.~H. 1994,
  \href{http://adsabs.harvard.edu/abs/1994A\%26AS..108..585S}{\JournalTitle{A\&AS},
  108, 585}

\bibitem[{{Semel}(1989)}]{thezdi}
{Semel}, M. 1989,
  \href{http://adsabs.harvard.edu/abs/1989A\%26A...225..456S}{\JournalTitle{A\&A},
  225, 456}

\bibitem[{{Stelzer} {et~al.}(2006){Stelzer}, {Micela}, {Flaccomio},
  {Neuh\"{a}user}, \& {Jayawardhana}}]{smf+06}
{Stelzer}, B., {Micela}, G., {Flaccomio}, E., {Neuh\"{a}user}, R., \&
  {Jayawardhana}, R. 2006,
  \href{http://dx.doi.org/10.1051/0004-6361:20053677}{\JournalTitle{A\&A}, 448,
  293}

\bibitem[{{Treumann}(2006)}]{t06}
{Treumann}, R. 2006,
  \href{http://dx.doi.org/10.1007/s00159-006-0001-y}{\JournalTitle{A\&A~Rev.},
  13, 229}

\bibitem[{{Vrba} {et~al.}(2004){Vrba}, {Henden}, {Luginbuhl}, {Guetter},
  {Munn}, {Canzian}, {Burgasser}, {Davy Kirkpatrick}, {Fan}, {Geballe},
  {Golimowski}, {Knapp}, {Leggett}, {Schneider}, \& {Brinkmann}}]{vhl+04}
{Vrba}, F.~J., {Henden}, A.~A., {Luginbuhl}, C.~B., {et~al.} 2004,
  \href{http://dx.doi.org/10.1086/383554}{\JournalTitle{AJ}, 127, 2948}

\bibitem[{{West} {et~al.}(2004){West}, {Hawley}, {Walkowicz}, {Covey},
  {Silvestri}, {Raymond}, {Harris}, {Munn}, {McGehee}, {Ivezi\'{c}}, \&
  {Brinkmann}}]{whw+04}
{West}, A.~A., {Hawley}, S.~L., {Walkowicz}, L.~M., {et~al.} 2004,
  \href{http://dx.doi.org/10.1086/421364}{\JournalTitle{AJ}, 128, 426}

\bibitem[{{Wu} \& {Lee}(1979)}]{theecm}
{Wu}, C.~S., \& {Lee}, L.~C. 1979,
  \href{http://dx.doi.org/10.1086/157120}{\JournalTitle{ApJ}, 230, 621}

\end{thebibliography}
