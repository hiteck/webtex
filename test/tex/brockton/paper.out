\BOOKMARK [1][-]{section*.1}{ABSTRACT}{section*.2}% 1
\BOOKMARK [1][-]{section.1}{Introduction}{}% 2
\BOOKMARK [1][-]{section.2}{Observations \046 Analysis}{}% 3
\BOOKMARK [1][-]{section.3}{Discussion and Conclusions}{}% 4
